src/main/scala/subsystem/Configs.scala

```diff
@@ -37,7 +37,8 @@ class BaseSubsystemConfig extends Config ((site, here, up) => {
     beatBytes = site(XLen)/8,
     blockBytes = site(CacheBlockBytes))
   // Additional device Parameters
-  case BootROMParams => BootROMParams(contentFileName = "./bootrom/bootrom.img")
+  case BootROMLocated(InSubsystem) => Some(BootROMParams(contentFileName = "./bootrom/bootrom.img"))
+  case SubsystemExternalResetVectorKey => false
```

Comment:
should BootROMLocated basically `require` that SubsystemExternalResetVectorKey` is false?

Reply:
Because I used a `BundleBroadcast` (i.e. a `BundleBridgeNexus` that `requires` that there only be a single bridge source input) the build will already fail if you don't set them compatibly.

Reply:
ok, that sounds good.